test/tests/test-helper.js

```diff
@@ -1,15 +1,12 @@
 import Schema from 'ember-orbit/schema';
 import Store from 'ember-orbit/store';
 
-var get = Ember.get,
-    set = Ember.set;
-
 var createStore = function(options) {
   options = options || {};
 
   var container = new Ember.Container();
```

Comment:
Someone more familiar with the changes here in 1.13 should rewrite this to not print deprecations... @dgeb ?